opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx

```diff
@@ -223,6 +223,7 @@ const LeftBar = () => {
     SETTINGS_SECURITYACTIVITY,
     SETTINGS_FILEINDEXING,
     SETTINGS_SUPPORT,
+    VIRTUAL_ORGANIZATION_ADMIN,
```

Comment:
why add this capability here ? organization admin case is handled line 904: 
`{isOrganizationAdmin && !isGrantedToSettings ? (`

Reply:
=> removed because unnecessary

Reply:
Refactor leftbar :
- Remove `VIRTUAL_ORGANIZATION_ADMIN` capability from `isGrantedToSecurity` to make the difference with `isOrganizationAdmin`.
- Remove `isGrantedToSecurity` because it is already checked by the Security component
- Remove the ternary operator to simplify & uniformize the Settings menu render.

@SouadHadjiat can you review this part, even if marie has already approuve ?